cpu/fe310/cpu.c

```diff
@@ -108,6 +111,18 @@ void cpu_init(void)
     /* Common RISC-V initialization */
     riscv_init();
 
+#ifdef MODULE_PMP_NOEXEC_RAM
+    /* This marks the complete RAM region from 0x80000000 to 0x80004000 as non
+     * executable. Using PMP entry 0.
+     *
+     * RAM starts at 2 GiB and is 16K in size
+     */
+    write_pmpaddr(0, make_napot(0x80000000, 0x4000));
```

Comment:
Don't we have defines for that?
```suggestion
    write_pmpaddr(0, make_napot(CPU_RAM_BASE, CPU_RAM_SIZE));
```

Now this is also no longer specific to `fe310`